### ISSUE TITLE
warning raised when using in an expo 45 project (react-native 0.68.2)

### ISSUE THREAD

**Issue by juuprz** [AndreiCalazans/rn-tooltip#92]
Warning message: ViewPropTypes will be removed from React Native. Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types'.

**Comment by AndreiCalazans** [comment]
Thanks! I'll update.

**Comment by ErikHasp** [comment]
+1, on Expo SDK 46 it actually throws errors so it's unusable 
```
Invariant Violation: ViewPropTypes has been removed from React Native. Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types'.
```

**Comment by AndreiCalazans** [comment]
I'll publish new version today without ViewPropTypes.

**Comment by avanavana** [comment]
Still getting this error.  Can't use rn-tooltips.
![Screen Shot 2022-08-30 at 2 58 54 PM](https://user-images.githubusercontent.com/5025683/187520491-46eeafb8-1da2-49a4-9583-c4296c3fd0ee.png)

**Comment by AndreiCalazans** [comment]
Should be fixed on 3.0.3 https://github.com/AndreiCalazans/rn-tooltip/releases/tag/v3.0.3
